sql/hive/src/test/scala/org/apache/spark/sql/hive/client/HadoopVersionInfoSuite.scala

```diff
@@ -35,39 +34,33 @@ class HadoopVersionInfoSuite extends SparkFunSuite {
   override protected val enableAutoThreadAudit = false
 
   test("SPARK-32256: Hadoop VersionInfo should be preloaded") {
-    val ivyPath =
-      Utils.createTempDir(namePrefix = s"${classOf[HadoopVersionInfoSuite].getSimpleName}-ivy")
```

Comment:
@LuciferYang, I think it's okay to use the temp dir to be extra safe. It speeds up but the downside is potential side-effect from the reused ivy. 1 min overhead should be okay

Reply:
OK, it seems that in the Intranet environment, I need to promote the Maven repository provider to improve the repository performance to solve this problem. The current Maven repository needs 6 hours to download the jars :(